utils/segment_logger.go

```diff
@@ -48,9 +50,9 @@ func GetLocalIP() string {
 	return "cannot get local ip"
 }
 
-func MapTransactionsToAddrs(txs []giota.Transaction) (addrs []giota.Address) {
+func MapTransactionsToAddrs(txs []transaction.Transaction) (addrs []trinary.Hash) {
 
-	addrs = make([]giota.Address, 0, len(txs))
+	addrs = make([]trinary.Hash, 0, len(txs))
```

Comment:
here and 2 lines above

Reply:
Sorry, this comment is not very clear to me. Can you explain?

Reply:
I was pointing out places where we should perhaps use the `Trytes` type instead of `Hash`, but disregard.  I did not know at the time that iota itself had decided to use the Hash type for addresses.